rep-0128.rst

```diff
@@ -37,7 +37,7 @@ This is the recommended layout for development is as follows:
           package.xml
         ...
         package_n/
-          CATKIN_IGNORE      --Optionally place this empty marker file to exclude package_n from being processed
+          CATKIN_IGNORE      --Optionally place this marker file to exclude package_n from being processed. Its file type (e.g. regular file, directory) and contents don't matter.
```

Comment:
How about a symlink and if the symlink points to an existing file / directory or not?

Reply:
I don't understand the question.

My thought on symlinks is that their referent should be ignored; just the name of the symlink file is taken into account. So you should be able to tell whether ament will ignore a directory using `ls -1` (dash one, not dash ell)

Reply:
I am fine that the presence of a symlink with that name is considered independently if the destination exists or not.

Please clarify that in the patch so it is clear how this is supposed to implemented. E.g. a simple `exists` check won't be sufficient if the symlink is pointing to a not existing destination.

Reply:
oooooh..... I understand now why you asked that. I didn't realize [`os.path.exists`](https://docs.python.org/3/library/os.path.html#os.path.exists) checks symlink validity; I assumed its behavior was that of [`os.path.lexists`](https://docs.python.org/3/library/os.path.html#os.path.lexists)

Reply:
Which patch do you want me to put the implementation details in?

Reply:
All patches which need it.